src/primevue/select/select.ts

```diff
@@ -0,0 +1,48 @@
+import { SelectPassThroughOptions } from "primevue/select";
+import { tw } from "@/lib/tags.ts";
+import "./select.css";
+
+export const base = tw`ris-body2-regular h-full min-h-48 w-full border-2 border-blue-800 bg-white px-16 py-4 outline-4 -outline-offset-4 outline-blue-800 placeholder:text-gray-600 hover:outline focus:outline disabled:border-blue-500 disabled:bg-white disabled:text-blue-500 disabled:outline-none [&+label]:my-4`;
+
+const select: SelectPassThroughOptions = {
+  root: ({}) => {
```

Comment:
Todo: Remove empty object pattern here and in other places in this file.

You might need to check your commit hooks locally, this is a linter error that should've been caught by ESLint on commit.